buildSrc/src/main/groovy/WaitForMySql.java

```diff
@@ -45,7 +45,11 @@ private void waitForConnection() {
       Connection connection = null;
       try {
 
-        String datasourceUrl = getenv("SPRING_DATASOURCE_URL", () -> "jdbc:mysql://localhost/eventuate?useSSL=true");
+        String datasourceUrl = getenv("SPRING_DATASOURCE_URL", () -> {
```

Comment:
WaitForMySQL even used?
It should all be replaced with Docker Compose PLugin

Reply:
ok, will do